debezium-core/src/main/java/io/debezium/pipeline/notification/channels/KafkaNotificationChannel.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.pipeline.notification.channels;
+
+import java.util.Map;
+
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.source.SourceRecord;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.debezium.config.CommonConnectorConfig;
+import io.debezium.function.BlockingConsumer;
+import io.debezium.pipeline.notification.Notification;
+import io.debezium.schema.SchemaFactory;
+import io.debezium.schema.SchemaNameAdjuster;
+
+public class KafkaNotificationChannel implements NotificationChannel, ConnectChannel {
```

Comment:
Could you please rename this class to `SinkNotificationChannel` and updated related code to it?

Reply:
Done.

Reply:
Why did you name it "**Sink**NotificationChannel"? I feel this makes an assumption (where it's hard to make an assumption) about the target, but the event has info from the source (source incr snapshot state). Furthermore, maybe some stream processing apps use this information, those aren't sinks, they transform or aggregate data in motion and those aren't sinks. Same goes for monitoring systems that use those events to show the status "realtime". Could this be used for other events too in the future? I feel the name is not the best considering what is inside those messages and how it can be used (yet and in the future).
Wdyt?

Reply:
@rk3rn3r Hi, the sink name was chosen to reflect the situation that the notofication goes out via Kafka Connect API, nothing more nothing less.

Reply:
This should be the name of the `output` channel. With the connect deployment this is Kafka but for debezium server this can be other message broker the we usually call `sink`.  I agree that it can be to generic, maybe `MessageBrokerNotificationChannel` can fit well?

Reply:
@mfvitale IMHO MessageBroker would be the same issue. And I want to avoud using `Connect*` based names as well. The only other generic name I can come up with is `DestinationNotificationChannel` but for me it is the same as sink.